config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   root "users#index"
-  get "/users/:id", to: "users#show"
-  get "/users/:id/posts", to: "users#posts"
-  get "/users/:id/posts/:id", to: "users#post"
+
+  resources :users, only: %i[index show] do
+    resources :posts, only: %i[index show create] do
```

Comment:
- [ ] After adding the new action, please don't forget that they should be a pair of routes for creating a post that's posts#new and posts#create(which is already available). The requirements of this project also asked us to do something of that sort.